internal/config/mount_config.go

```diff
@@ -50,6 +54,15 @@ const (
 	DefaultGrpcConnPoolSize                       = 1
 	DefaultAnonymousAccess                        = false
 	DefaultEnableHNS                              = false
+
+	// MetadataPrefetchModeDisabled is the mode without metadata-prefetch.
+	MetadataPrefetchModeDisabled metadataPrefetchModeValue = "disabled"
```

Comment:
Should this be a child element if metadata-cache? IIUC, the effort is to pre-populate the metadata cache.